src/main/java/pacApp/pacSecurity/CustomCorsFilter.java

```diff
@@ -20,7 +20,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         HttpServletResponse response = (HttpServletResponse) servletResponse;
         response.setHeader("Access-Control-Allow-Origin", "*");
         response.setHeader("Access-Control-Allow-Methods", "GET,POST,PUT,DELETE,OPTIONS");
-        //response.setHeader("Access-Control-Allow-Headers", "*");
+        response.setHeader("Access-Control-Allow-Headers", "*");
```

Comment:
This values are not acceptable for production. Could we replace the wildcards with the correct headers?